android/assets/jsons/translations/Portuguese.properties

```diff
@@ -722,97 +722,97 @@ You have won a scientific victory! = Você ganhou uma vitória ciêntifica!
+Their citizens generate 2x the unhappiness, unless you build a courthouse. = Seus cidadões geram 2x a infelicidade, a não ser que você construa um tribunal.
  # Requires translation!
-Puppet = 
+Puppet = Fantoche
  # Requires translation!
-Puppeted cities do not increase your tech or policy cost, but their citizens generate 1.5x the regular unhappiness. = 
+Puppeted cities do not increase your tech or policy cost, but their citizens generate 1.5x the regular unhappiness. = Cidades fantoche não aumentam seus custos de tecnologia ou politica, mas seus cidadões geram 1,5x infelicidade normal.
  # Requires translation!
-You have no control over the the production of puppeted cities. = 
+You have no control over the the production of puppeted cities. = Você não tem controle sobre a prodção das cidades fantoche.
  # Requires translation!
-Puppeted cities also generate 25% less Gold and Science. = 
+Puppeted cities also generate 25% less Gold and Science. = Cidades fantoche também geram 25% a menos de Ouro e Ciência.
  # Requires translation!
-A puppeted city can be annexed at any time. = 
+A puppeted city can be annexed at any time. = Uma cidade fantoche pode ser anexada a qualquer momento.
  # Requires translation!
-Liberate = 
+Liberate = Liberar
  # Requires translation!
-Liberating a city returns it to its original owner, giving you a massive relationship boost with them! = 
+Liberating a city returns it to its original owner, giving you a massive relationship boost with them! = Liberar uma cidade devolve ela a seu dono original, dando um estímulo gigantesco de relacionamento com ele!
  # Requires translation!
-Raze = 
+Raze = Arrasar
  # Requires translation!
-Razing the city annexes it, and starts razing the city to the ground. = 
+Razing the city annexes it, and starts razing the city to the ground. = Arrasar uma cidade a anexa, e começa a arrasa-la às cinzas.
  # Requires translation!
-The population will gradually dwindle until the city is destroyed. = 
+The population will gradually dwindle until the city is destroyed. = A população ira gradualmente diminuir até que a cidade seja destruída.
  # Requires translation!
-Destroy = 
+Destroy = Destruir
  # Requires translation!
-Destroying the city instantly razes the city to the ground. = 
+Destroying the city instantly razes the city to the ground. = Destruir a cidade arrasa instantaneamente a cidade às cinzas.
  # Requires translation!
-Remove your troops in our border immediately! = 
+Remove your troops in our border immediately! = Remova suas tropas das nossas fronteiras imediatamente!
 Sorry. = Desculpe.
 Never! = Nunca!
 
  # Requires translation!
-Offer Declaration of Friendship ([30] turns) = 
+Offer Declaration of Friendship ([30] turns) = Oferecer Declaração de Amizade ([30] turnos)
  # Requires translation!
-My friend, shall we declare our friendship to the world? = 
+My friend, shall we declare our friendship to the world? = Meu amigo, deveríamos declarar nossa maizade ao mundo?
```

Comment:
Looks like a letter-switch typo

Reply:
Yes, you are correct.
It should be 'amizade'